README.md

```diff
@@ -152,7 +152,7 @@ Also, setting values that contain colons are parsed as list.
 
 `ARTIFACT_EXCLUDE_PATTERNS` Specifies a list of glob patterns that can be used to
 exclude files from being recorded as materials or products. See [runlib
-docs for more details](https://github.com/in-toto/in-toto/blob/develop/in_toto/runlib.py#L93-L114).
+docs for more details](https://github.com/in-toto/in-toto/blob/develop/in_toto/runlib.py#L101-L119).
```

Comment:
Since you're already at it, please replace the absolute link, which points to `runlib.py` in the `develop` branch, with a relative link:
`[runlib docs for more details](in_toto/runlib.py#L101-L119)`? 

Please also update the line numbers so that they match your changes below.